server/lib/topic.js

```diff
@@ -61,13 +63,15 @@ const channels = {
 // writes events into the event database table
 function databaseLogger(data) {
 
-  let record = {
+  const record = {
     timestamp: new Date(data.timestamp),
     user_id: data.user_id,
     channel: data.channel,
+    entity : data.entity.toUpperCase(),
```

Comment:
Is entity conceptually required to do publish an update? 

If data doesn't have an `entity` attribute it looks like this would throw an uncaught exception.